src/app/learn/get-started/advanced/accessibility-unit-tests/index.html

```diff
@@ -0,0 +1,145 @@
+<stache
+  pageTitle="Accessibility Tests"
+  navTitle="Accessibilty tests"
+  showTableOfContents="true"
+  navOrder="1">
+  
+  <stache-page-summary>
+    <p>
+      The <a href="https://github.com/blackbaud/skyux-lib-testing">SKY UX testing library</a> provides methods to run <a href="https://github.com/dequelabs/axe-core">aXe automated accessibility tests</a> in your unit tests. aXe is an open source library of automated accessibility rules that is integrated into SKY UX to identify potential issues for a subset of the <a href="https://www.w3.org/TR/WCAG20/">Web Content Accessibility Guidelines 2.0 Level A and AA success criteria</a>. For SKY UX guidance on designing, coding, and testing accessible experiences, see the accessibility section.
+    </p>
+  </stache-page-summary>
+
+  <p>
+    <stache-include fileName="e2e-recommendation.html"></stache-include> If you must use e2e tests, the deprecated accessibilty analyzer lets you add accessibility checks to e2e specs and manage rules for accessibility tests with an <stache-code>a11y</stache-code> configuration property. For more information, see <a routerLink="/learn/get-started/advanced/accessibility-analyzer">the accessibility analyzer</a> and <a routerLink="/learn/reference/configuration">the configuration guidelines</a>.
+  </p>
+  
+  <stache-page-anchor>
+    Import SKY UX testing library
+  </stache-page-anchor>
+  
+  <p>
+    The first step to run accessibility checks in SKY UX unit tests is to import the <a href="https://github.com/blackbaud/skyux-lib-testing">SKY UX testing library</a> into your SPA. This library provides methods to interact with SKY UX during Karma unit tests. For accessibility testing, it extends the Jasmine <stache-code>expect</stache-code> function to add the <stache-code>toBeAccessible</stache-code> matcher.
+  </p>
+  <p>
+    To install the testing library, open the command prompt, <stache-code>cd</stache-code> into your SPA's directory, and run:
+  </p>
+  <stache-code-block>
+    npm install @blackbaud/skyux-lib-testing --save-dev --save-exact
+  </stache-code-block>
+  <p>
+    By default, the <stache-code>skyA11yAnalyzerConfig</stache-code> interface specifies <a href="https://github.com/blackbaud/skyux-lib-testing/blob/master/src/app/public/src/a11y/a11y-analyzer.ts">configuration settings for <stache-code>toBeAccessible</stache-code> to indicate which accessibility checks to run</a>. To overwrite the default acccessibility checks, you can modify <a stacheRouterLink="." fragment="specify-parameters-for-tobeaccessible">a parameter for the <stache-code>toBeAccessible</stache-code> matcher</a>.
```

Comment:
I believe `SkyA11yAnalyzerConfig` should be capitalized. (This appears throughout the docs)

Reply:
Capitalized `SkyA11yAnalyzerConfig` throughout.